test/test_config.js

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2014 Open Ag Data Alliance
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+
+module.exports = {
+  server: {
+    jsonSpaces: 2,
+    sessionSecret: 'Xka*32F@*!15',
+    passwordSalt: '$2a$10$l64QftVz6.7KR5BXNc29IO',
+    port: 8443,
+    httpMode: false,
+    root: 'https://localhost:8443',
```

Comment:
@abalmos changed the `config.js` format slightly. Perhaps updating yours to match the new format will make things work?

`httpMode` and `root` have been replaced by new things (see `config.js`).